internal/pkg/cli/task_run_test.go

```diff
@@ -214,7 +214,7 @@ func TestTaskRunOpts_Validate(t *testing.T) {
 					Name: "my-app",
 				}, nil)
 			},
-			wantedError: errors.New("get environment: couldn't find environment dev in the application my-app"),
+			wantedError: fmt.Errorf("get environment %s config: couldn't find environment dev in the application my-app", "dev"),
```

Comment:
I think I'd prefer not using the formatted string when it comes to unit test, since we don't pass in any string variable as params.